src/rabbit.erl

```diff
@@ -287,7 +287,7 @@ broker_start() ->
     case code:load_file(sd_notify) of
         {module, sd_notify} -> SDNotify = sd_notify,
                                SDNotify:sd_notify(0, "READY=1");
-        {error, _} -> ok
+        {error, _} -> os:cmd("systemd-notify --ready")
```

Comment:
Thinking out loud: `os:cmd/1` always returns a string, even when execution failed. I couldn't make it throw an exception, so this is hopefully safe.